src/models/MediaImage.ts

```diff
@@ -10,9 +10,10 @@ import type { DraftL } from './Draft'
 import type { StepL } from './Step'
 import type { MouseEvent } from 'react'
 
-import { existsSync, mkdirSync, readFileSync } from 'fs'
+import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'fs'
 import Konva from 'konva'
 import { lookup } from 'mime-types'
+import { format, join, parse } from 'path'
```

Comment:
every `path` import should use `pathe` instead, to ensure more consistency across OSs